docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+services:
+
+  inginious-db:
+    image: mongo:6.0.2
+    networks:
+      - inginious
+
```

Comment:
inginious-base seems missing from here. Don't know if this is intentional.

Reply:
This is the common layer between all the containers. It is not meant to run by itself.